Gemfile.lock

```diff
@@ -499,38 +502,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
+    parallel (1.24.0)
+    parallel_tests (2.32.0)
       parallel
     parser (2.7.2.0)
       ast (~> 2.4.1)
-    path_expander (1.0.4)
-    pg (1.1.3)
-    pry (0.10.3)
-      coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
+    path_expander (1.1.1)
+    pg (1.5.4)
+    pry (0.14.2)
```

Comment:
### [pry](http://pry.github.io)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.10.3 → 0.14.2

[_[change-log](https://github.com/pry/pry/blob/master/CHANGELOG.md), [source-code](https://github.com/pry/pry), [gem-diff](https://my.diffend.io/gems/pry/0.10.3/0.14.2)_]





<details>
<summary>Commits</summary>

A change of **1206** commits. See the full changes on [the compare page](https://github.com/pry/pry/compare/v0.10.3...v0.14.2).

These are the first 10 commits:
- (03afca9) [Merge pull request #2021 from pry/config-influence-reduction](https://github.com/pry/pry/commit/03afca9eafe4f2981edad1cfe4346a643fb64d72)
- (e5556a2) [Refactor Config](https://github.com/pry/pry/commit/e5556a2be8627ec3fe594c738f10422d5a1f5d43)
- (e2c4aa6) [Merge pull request #2024 from pry/configuration-refactoring](https://github.com/pry/pry/commit/e2c4aa680c2876f3cc8ffd1724c639e0a6a74148)
- (c274756) [history: cache @history.count to improve performance](https://github.com/pry/pry/commit/c2747569bb1a16757f6d81f31d74d272f1dafa84)
- (ed45fb3) [Merge pull request #2025 from pry/2007-performance-slowdown-…](https://github.com/pry/pry/commit/ed45fb3087029f82d9a93497100b35235dd13335)
- (61fa19b) [commands/show_info: fix formatting error for C method header](https://github.com/pry/pry/commit/61fa19b1f04511173abb70f3d1814f03ed2d6a13)
- (16fd61b) [Merge pull request #2026 from pry/2006-show-source-formattin…](https://github.com/pry/pry/commit/16fd61be2ff344e0105af1764c6e6270bb393755)
- (d80c651) [config: add `rc_file` that allows specifying `pryrc` file](https://github.com/pry/pry/commit/d80c6517d4a601cb5eb3c2a6dfdb802f5e683a68)
- (6f3cd93) [Merge pull request #2027 from pry/rc-file-attribute](https://github.com/pry/pry/commit/6f3cd9310125403c6e5d78832553df8bbee99e46)
- (e7de859) [pry_instance: fix Style/AccessModifierDeclarations cop offen…](https://github.com/pry/pry/commit/e7de85913ce72b6724d914801304256d2084286f)


</details>